auth/server/src/main/java/com/seagates3/controller/AccountController.java

```diff
@@ -176,7 +179,23 @@ public ServerResponse create() {
 
         AccessKey rootAccessKey;
         try {
-            rootAccessKey = createRootAccessKey(root);
+            rootAccessKey = accessKeyDAO.find(accessKey);
```

Comment:
In case if system generated key, You will search LDAP with NULL key here
Make sure it will not fail with exception or put NULL check otherwise

Reply:
Will fix it.

Reply:
Done

Reply:
cant see this fixed

Reply:
null check was missed in the commit. Added it.
Done